sdk/appconfiguration/app-configuration/review/app-configuration.api.md

```diff
@@ -37,6 +38,7 @@ export class AppConfigurationClient {
 
 // @public
 export interface AppConfigurationClientOptions {
+    retryOptions?: Pick<RetryOptions, "maxRetries" | "maxRetryDelayInMs">;
```

Comment:
I believe in other clients this is part of the options in the client methods, instead of the constructor 🤔

Reply:
Not for Service Bus at least.

Reply:
Not for Storage libraries too.

Reply:
Not for Keyvault libraries as well.

Reply:
Can you provide an example?

Reply:
I wonder whether it's better to define the interface here instead of depending on core-http so moving to core v2 in the future could be easier?

Reply:
+1 to redefining the interface locally

Reply:
I don't mind, will update.

Reply:
Updated.

Reply:
@HarshaNalluru thank you for the clarification! I’ll resolve this